Tests/YBottomSheetTests/BottomSheetControllerTests.swift

```diff
@@ -324,6 +290,56 @@ final class BottomSheetControllerTests: XCTestCase {
     }
 }
 
+extension BottomSheetControllerTests {
```

Comment:
tests should not be in an extension. Only helper methods. If SwiftLint is complaining, it's ok to disable it at the top of the file with a comment. e.g. in YMatterType test files we have:
```
// It's not a problem having too many tests!
// swiftlint:disable file_length
// swiftlint:disable type_body_length
```

Reply:
So please put these back where they were.

Reply:
Sure Mark